src/app/controllers/campaigns/campaign.show.js

```diff
@@ -1,10 +1,10 @@
 angular.module('crowdFundApp').controller('campaignShow',
-  function($scope, CampaignService, $sce, $auth, $stateParams) {
+  function($scope, CampaignService, $sce, $auth, $stateParams, Upload) {
 
     CampaignService.getCampaign().then(
       function(success) {
         $scope.campaign = success.data
-
+        console.log($scope.campaign)
```

Comment:
Please don't push debugging/logging code, unless it's logging the application behaviour itself.